pkg/webui/console/views/admin-user-management-add/index.js

```diff
@@ -49,7 +49,7 @@ import {
     getConfiguration: () => getIsConfiguration(),
   },
 )
-@withRequest(({ getConfiguration }) => getConfiguration(), ({ fetching }) => fetching)
+@withRequest(({ getConfiguration }) => getConfiguration())
```

Comment:
The `error`\-prop is not selected in the `connect()` above. The `withRequest` will then not be aware of any errors that the action might have resulted in.